### DISCUSSION TITLE
Does toga.Selection have fuzzy matching function in the android app?

### DISCUSSION THREAD

**Discussion by ccinsz** [beeware/briefcase#1493] [Q&A]
Does toga.Selection have fuzzy matching function in the android app? If has, how to make it.

**Comment by freakboy3742** [comment]
You'll need to elaborate on what you mean. A selection widget is a list of fixed options from which the user can select. What does "fuzzy matching" mean in this context? And why is it Android specific?

**Reply by ccinsz** [reply]
For example, when there are 100 options in the  a selection widget,  it takes a long time to select one. I would like to know if there is a function that can input certain characters and the selection widget will display matching items.

And why is it Android specific?
Because under Windows, after clicking a selection widget and inputting certain characters on the keyboard, it will automatically jump to the corresponding item.

**Reply by freakboy3742** [reply]
Toga is exposing the native Winforms selection widget on Windows, and the native Android selection widget on Android. I'm not aware of any analog of Windows' keyboard-based navigation aid on Android - if only because Android doesn't have a keyboard visible when the selection widget is visible. 

If there *is* a native Android analog of the Winforms behavior, we'd be happy to consider adding it.
